lua/rocks-dev/init.lua

```diff
@@ -7,13 +7,21 @@ function rocks_dev.setup(user_configuration)
         return
     end
 
+    local dev_path = user_configuration.dev and user_configuration.dev.path
+
     for _, data in pairs(user_configuration.plugins or {}) do
+        local path
         if data.dir then
-            vim.opt.runtimepath:append(vim.fn.expand(data.dir))
+            path = vim.fn.expand(data.dir)
+        elseif dev_path and data.dev then
+            path = vim.fn.expand(vim.fs.joinpath(dev_path, data.name))
+        end
```

Comment:
question: Do you think it would make sense to be able to set both `<plugin>.dev = true` and `<plugin>.dir = "<subdirectory>"`, in which case it would be located in e.g. `~/Projects/<subdirectory>`?

If this isn't allowed, setting both `dir` and `dev` + `dev_path` would be an invalid state.

Reply:
I think any behavior is fine. I made `dir` take priority so `dev` is ignored if `dir` is present, but I didn't document that.

Whatever you would prefer, I can do that and I'll make sure to document it.

Reply:
This is fine :+1: 

Come to think of it,  my suggestion may be overly complex and could lead to surprises.
We can always add that capability later if people need it.